sdk/cognitiveservices/cognitiveservices-luis-runtime/README.md

```diff
@@ -17,124 +17,143 @@ npm install @azure/cognitiveservices-luis-runtime
       resolution: {}
-    }],
-    dynamicLists: [{
+    }
+  ],
+  dynamicLists: [
+    {
       listEntityName: "testlistEntityName",
-      requestLists: [{
-        name: "testname",
-        canonicalForm: "testcanonicalForm",
-        synonyms: ["testsynonyms"]
-      }]
-    }]
-  };
-  const verbose = true;
-  const showAllIntents = true;
-  const log = true;
-  client.prediction.getVersionPrediction(appId, versionId, predictionRequest, verbose, showAllIntents, log).then((result) => {
+      requestLists: [
+        {
+          name: "testname",
+          canonicalForm: "testcanonicalForm",
+          synonyms: ["testsynonyms"]
+        }
+      ]
+    }
+  ]
+};
+const verbose = true;
+const showAllIntents = true;
+
+client.prediction
+  .getVersionPrediction(appId, versionId, predictionRequest, { verbose, showAllIntents })
+  .then((result) => {
     console.log("The result is:");
     console.log(result);
+  })
+  .catch((err) => {
+    console.error(err);
   });
-}).catch((err) => {
-  console.error(err);
-});
 ```
 
 #### browser - Authentication, client creation and getVersionPrediction prediction as an example written in JavaScript.
 
-##### Install @azure/ms-rest-browserauth
+##### Install @azure/ms-rest-azure-js
 
 ```bash
-npm install @azure/ms-rest-browserauth
+npm install @azure/ms-rest-azure-js
 ```
 
 ##### Sample code
 
-See https://github.com/Azure/ms-rest-browserauth to learn how to authenticate to Azure in the browser.
-
 - index.html
+
 ```html
 <!DOCTYPE html>
 <html lang="en">
   <head>
     <title>@azure/cognitiveservices-luis-runtime sample</title>
-    <script src="node_modules/@azure/ms-rest-js/dist/msRest.browser.js"></script>
-    <script src="node_modules/@azure/ms-rest-browserauth/dist/msAuth.js"></script>
+    <script src="node_modules/@azure/ms-rest-azure-js/dist/cognitiveServicesCredentials.js"></script>
     <script src="node_modules/@azure/cognitiveservices-luis-runtime/dist/cognitiveservices-luis-runtime.js"></script>
     <script type="text/javascript">
-      const subscriptionId = "<Subscription_Id>";
-      const authManager = new msAuth.AuthManager({
-        clientId: "<client id for your Azure AD app>",
-        tenant: "<optional tenant for your organization>"
-      });
-      authManager.finalizeLogin().then((res) => {
-        if (!res.isLoggedIn) {
-          // may cause redirects
-          authManager.login();
-        }
-        const client = new Azure.CognitiveservicesLuisRuntime.LUISRuntimeClient(res.creds, subscriptionId);
-        const appId = ec7b1657-199d-4d8a-bbb2-89a11a42e02a;
-        const versionId = "testversionId";
-        const predictionRequest = {
-          query: "testquery",
-          options: {
-            datetimeReference: new Date().toISOString(),
-            overridePredictions: true
-          },
-          externalEntities: [{
+      let authoringKey = process.env["luis-authoring-key"];
+      const creds = new CognitiveServicesCredentials(authoringKey);
```

Comment:
Since the `@azure/ms-rest-azure-js` package does not have a browser bundle yet, we should use the `ApiKeyCredentials` from `@azure/ms-rest-js` instead.

```suggestion
      const creds = new ApiKeyCredentials({ inHeader: { 'Ocp-Apim-Subscription-Key': authoringKey } });
```